tests/nhdp/Makefile

```diff
@@ -0,0 +1,12 @@
+APPLICATION = nhdp
+include ../Makefile.tests_common
+
+BOARD_BLACKLIST := arduino-mega2560 msb-430 msb-430h telosb wsn430-v1_3b wsn430-v1_4 z1
+BOARD_INSUFFICIENT_MEMORY := nucleo-f334 stm32f0discovery weio
+
+USEMODULE += gnrc_ipv6
+USEMODULE += gnrc_conn_udp
+USEMODULE += nhdp
+USEPKG += oonf_api
```

Comment:
Not necessary (and leads to errorneous output: `/home/martine/Repositories/RIOT-OS/RIOT/tests/nhdp/../../Makefile.include:261: target '/home/martine/Repositories/RIOT-OS/RIOT/tests/nhdp/bin/native/oonf_api.a' given more than once in the same rule`)